doc/api_rstgen.py

```diff
@@ -80,13 +129,15 @@ def _generate_api_index_rst_files():
         folder_index = _get_file_path(folder, f"{folder}_contents")
         with open(folder_index, "w", encoding="utf8") as index:
             index.write(f".. _ref_{folder}:\n\n")
-            index.write(f"{folder}\n")
+            index.write(
+                "API reference\n" if folder == "visualization" else f"{folder}\n"
```

Comment:
Instead of hardcoding and handling this here, we can add it in `_write_rst_file` as done in PyFluent as well:

https://github.com/ansys/pyfluent/blob/main/doc/api_rstgen.py#L18